bot/start.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env sh
+# startup script for wuestbot on gunicorn
+
+# Project Name
+NAME="wuestbot"
+
+# Host to listen on
+HOST=127.0.0.1
+
+# Port to listen on
+PORT=8443
+
+# Number of gunicorn Workers
+NUM_WORKERS=2
+
+# Databank directory
+DB_DIR=app/database
+
+# create database
+source $DB_DIR/db_create.py
```

Comment:
Actually I would not include this here and call the setup separately. For example if we call `start.sh` as the primary command to start the bot, and container/process crashes and restart, `db_create.py` would get called again which we don't want.

Reply:
this is why i used try/except clauses for table creation in `db_create.py`. But might change those clauses to 'pure' SQL like `CREATE TABLE ralph IF NOT EXISTS` ?

Reply:
`CREATE TABLE ralph IF NOT EXISTS` is the cleaner way I think

Reply:
fixed